io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+        sha256: 56e7e64ed86d7d9ec59500fd34f26f881bdb9d541916d9a817c3bfb8cf0e9508
+    buildsystem: autotools
+    config-opts:
+      - --enable-ccp4
+      - --enable-shared
+      - --disable-static
+    build-options:
+      cflags: "-fPIC"
+    build-commands:
+      - ./configure --prefix=/app --enable-ccp4 --enable-shared --disable-static
+      - make
+      - make install
+
+  - name: boost
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --with-icu=/app --prefix=/app --libdir=/app/lib
+      - ./b2 install --prefix=/app --libdir=/app/lib -d2 -j$(nproc) threading=multi link=shared,static
+    config-opts:
+      - --without-libraries=[python, mpi]
+    build-options:
+      env:
+        CXXFLAGS: "-std=c++14"
+    cleanup:
+      - /app/share/boost
+
+  - deps/pypi-dependencies.json
+
+  - name: gobject-introspection
+    buildsystem: meson
+    config-opts:
+      - -Dpython=/usr/bin/python3
+    builddir: true
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
+        sha256: 0f5a4c1908424bf26bc41e9361168c363685080fbdb87a196c891c8401ca2f09
+    post-install:
+      - ln -sf /app/libexec/bin/* /app/bin/
+
+  - name: boost-python3
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+    buildsystem: simple
+    build-commands:
+      - echo "using python" > user-config.jam
+      - echo ": 3.12" >> user-config.jam
+      - echo ": /usr/bin/python3" >> user-config.jam
+      - echo ": /usr/include/python3.12" >> user-config.jam
+      - echo ": /usr/lib" >> user-config.jam
+      - echo ";" >> user-config.jam
+      - ./bootstrap.sh --prefix=/app --libdir=/app/lib --with-python=python3.12 --with-python-root=/usr --with-libraries=python
+      - ./b2 --build-dir=build-python3 --stagedir=stage-python3 --prefix=/app --libdir=/app/lib abi=sysv linkflags=-L/usr/lib --user-config=user-config.jam python=3.12 -d2 -j$(nproc) --layout=system threading=multi link=shared,static install
+    build-options:
+      env:
+        CXXFLAGS: "-std=c++14"
+        PYTHON: "/usr/bin/python3"
+        PYTHONINCLUDE: "/usr/include/python3.12"
+        PYTHONLIB: "/usr/lib"
+    cleanup:
+      - /app/share/boost
+
+  - name: libdwarf
+    sources:
+      - type: archive
+        url: https://www.prevanders.net/libdwarf-0.11.0.tar.xz
+        sha256: 846071fb220ac1952f9f15ebbac6c7831ef50d0369b772c07a8a8139a42e07d2
+    buildsystem: simple
+    build-options:
+      environment:
```

Comment:
here and everywhere else `environment` does exist. That mean you can remove it everywhere.